### PR TITLE
[sdk 326] Hours, Day, time zone added with every request

### PR DESCRIPTION
**Pull request by ZahidZafar**

### CODE REVIEW

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Why are you removing those "if" statements? You are breaking functionality and introducing bugs with that.

**Comment by ZahidZafar** [comment]
when ever this get call, null is passing to timestamp,
we even can remove this param .
and more over fresh timestamp should added with base request.

**Comment by ArtursKadikis** [comment]
Not with this PR.

File: countlyCommon/countlyCommon/Helpers/TimeHelper.cs [hunk 1]
```diff
@@ -41,5 +41,18 @@ public static long UnixTimeNow()
...
+        /// <summary>
+        ///Convert a Unix timestamp to DateTime
+        /// </summary>
+        /// <param name="unixTimeStamp"></param>
+        /// <returns>DateTime</returns>
+        internal static DateTime UnixTimeStampToDateTime(long? unixTimeStamp)
+        {
+            // Unix timestamp is seconds past epoch
+            DateTime dtDateTime = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
+            dtDateTime = dtDateTime.AddSeconds((double)unixTimeStamp).ToLocalTime();
```
Comment (ArtursKadikis) [inline countlyCommon/countlyCommon/Helpers/TimeHelper.cs:54]: Bug. Converting to "double" from "long" leads to loss of data. Rework this

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

File: countlyCommon/countlyCommon/Helpers/TimeHelper.cs (hunk 1 above)
Comment (ZahidZafar) [reply countlyCommon/countlyCommon/Helpers/TimeHelper.cs:54]: > Bug. Converting to "double" from "long" leads to loss of data. Rework this
SDK already converting double to long, m casting this long to double back.
well! i am going o remove this function.

image

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Add a changelog entry that hour, day of week and timezone information will now be added to requests.

**Pull request merged**